### PR TITLE
merge formal snapshot and snapshots docs

### PR DESCRIPTION
**Pull request by johnjmartin**
## Description 

most of the info in the formal snapshot page is stale, this merges the two docs

### CODE REVIEW

File: docs/content/guides/operator/snapshots.mdx [hunk 1]
```diff
@@ -60,8 +70,9 @@ Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 se
 
 Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:
    ```yaml
-   state-snapshot-write-config:
-      object-store-config:
+   db-checkpoint-config:
+     state-snapshot-write-config:
+       object-store-config:
```
Comment (williampsmith) [inline docs/content/guides/operator/snapshots.mdx:75]: When did we make this change?

File: docs/content/guides/operator/snapshots.mdx [hunk 2]
```diff
@@ -25,6 +25,16 @@ Formal snapshots are not suitable for use if you are running an RPC node that do
...
+1. **Verifiability:** Formal snapshots have first-class support in the core Sui protocol. As such, they must be trustless/verifiable by the node operator upon download. To support this, the protocol signs a commitment to the live object state at end of epoch, which formal snapshots are checked against at restore time. If this verification fails, the node state is moved back to the state before the snapshot restore attempt.
+
+Because these snapshots do not contain indexes, they are most immediately useful for validators and state sync Full nodes (SSFNs). You can upload snapshots to remote object stores like S3, Google Cloud Storage, Azure Blob Storage, and similar services. These services typically run the export process in the background so there is no degradation in performance for your Full node. With snapshots stored in the cloud, you can recover from catastrophic failures in your system or hardware more efficiently.
+
 ## Enabling snapshots
 
 Full nodes do not take snapshots by default. To enable this feature you must apply specific configs to your Full node.
```
Comment (williampsmith) [inline docs/content/guides/operator/snapshots.mdx:40]: Not sure why it won't let me write a comment below, but on line 48 can we rename to `### Enabling DB snapshots` for consntsency?

Comment (williampsmith) [reply docs/content/guides/operator/snapshots.mdx:40]: Also what are your thoughts on changing line 2 above to be `DB Snapshots` or `Database Snapshots` or `Full Snapshots`? I think part of the confusion has been the fact that "snapshots" is now ambiguous, so we should disambiguate in all cases, not just some.

**Review by williampsmith — APPROVED** [review]

File: docs/content/guides/operator/snapshots.mdx (hunk 1 above)
Comment (johnjmartin) [reply docs/content/guides/operator/snapshots.mdx:75]: > Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:

I was just looking at the directive above this config block - should it not be nested?

Comment (johnjmartin) [reply docs/content/guides/operator/snapshots.mdx:75]: Checking your docs, I think this was just a bad copy paste that was made earlier - I'll fix

Comment (williampsmith) [reply docs/content/guides/operator/snapshots.mdx:75]: Sounds good. Yeah `state-snapshot-write-config` should be a global config key in the node config: https://github.com/MystenLabs/sui/blob/e1ca530f346ebc13816472a65429448843ec5508/crates/sui-config/src/node.rs#L146

**Pull request merged**